aospy/test/test_roundtrip.py

```diff
@@ -184,5 +198,89 @@ def test_monthly_ts(self):
     def test_vert_int_pressure(self):
         pass
 
+
+@unittest.skipIf(not model_files_exist(am3), skip_message)
+class TestAM3(TestAM2):
+    def setUp(self):
+        self.olr_test_params = {'proj': aospy_test,
+                                'model': am3,
+                                'run': test_am3,
+                                'var': olr,
+                                'date_range': ('1981-01-01', '2010-12-31'),
+                                'intvl_in': 'monthly',
+                                'dtype_in_time': 'ts',
+                                'dtype_in_vert': 'pressure',
+                                'dtype_out_vert': False,
+                                'level': False}
+        self.temp_test_params = {'proj': aospy_test,
+                                 'model': am3,
+                                 'run': test_am3,
+                                 'var': temp,
+                                 'date_range': ('1981-01-01',
+                                                '2010-12-31'),
+                                 'intvl_in': 'monthly',
+                                 'dtype_in_time': 'ts',
+                                 'level': False}
+
+
+@unittest.skipIf(not model_files_exist(hiram), skip_message)
+class TestHiRAM(TestAM2):
+    def setUp(self):
+        self.olr_test_params = {'proj': aospy_test,
+                                'model': hiram,
+                                'run': test_hiram,
+                                'var': olr,
+                                'date_range': ('1979-01-01', '1995-12-31'),
+                                'intvl_in': 'monthly',
+                                'dtype_in_time': 'ts',
+                                'dtype_in_vert': 'pressure',
+                                'dtype_out_vert': False,
+                                'level': False}
+        self.temp_test_params = {'proj': aospy_test,
+                                 'model': hiram,
+                                 'run': test_hiram,
+                                 'var': temp,
+                                 'date_range': ('1979-01-01',
+                                                '1995-12-31'),
+                                 'intvl_in': 'monthly',
+                                 'dtype_in_time': 'ts',
+                                 'level': False}
+
+    @unittest.skip(('could not immediately find a case with temperature'
+                    ' output on sigma levels in CMIP5 archive'))
+    def test_vert_int_sigma(self):
+        pass
+
+
+# cesm1_cam5 has no grid files so just use hiram's to test to skip
+@unittest.skipIf(not model_files_exist(hiram), skip_message)
```

Comment:
Here's where I'd add the expected failure decorator.  I think using both will work -- we want to keep the skip decorator, for testing not on the GFDL machines, i.e. we want to skip if files aren't there and expected failure if files are there.